pkg/sql/colexec/generate_series/generate_series.go

```diff
@@ -0,0 +1,282 @@
+// Copyright 2022 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package generate_series
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/matrixorigin/matrixone/pkg/common/moerr"
+	"github.com/matrixorigin/matrixone/pkg/container/batch"
+	"github.com/matrixorigin/matrixone/pkg/container/types"
+	"github.com/matrixorigin/matrixone/pkg/container/vector"
+	"github.com/matrixorigin/matrixone/pkg/sql/plan"
+	"github.com/matrixorigin/matrixone/pkg/vm/process"
+	"math"
+	"strconv"
+	"strings"
+)
+
+func String(arg any, buf *bytes.Buffer) {
+	buf.WriteString("generate_series")
+}
+
+func Prepare(_ *process.Process, arg any) error {
+	return nil
+}
+
+func Call(_ int, proc *process.Process, arg any) (bool, error) {
+	param := arg.(*Argument).Es
+	bat := proc.InputBatch()
+	if bat == nil {
+		return true, nil
+	}
+	dt := bat.GetVector(0).GetBytes(0)
+	var exArgs []string
+	err := json.Unmarshal(dt, &exArgs)
+	if err != nil {
+		return false, err
+	}
+	rbat := batch.New(false, param.Attrs)
```

Comment:
add 'rbat.Cnt = 1' here,
then rbat.Clean() can release the memory.

and if err happens, you should clean the rbat.